lib/redis_memo/future.rb

```diff
@@ -37,15 +37,23 @@ def method_cache_key
       RedisMemo::MemoizeMethod.__send__(:method_cache_keys, [context])&.first || ''
   end
 
+  def validate_cache_result
+    cache_validation_sample_percentage =
+      @cache_options[:cache_validation_sample_percentage]&.to_i || RedisMemo::DefaultOptions.cache_validation_sample_percentage&.to_i
```

Comment:
```suggestion
      @cache_options[:cache_validation_sample_percentage] || RedisMemo::DefaultOptions.cache_validation_sample_percentage
```
no need for those to_i calls. if it fails it's users fault